src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  flex-wrap: wrap;
+  padding: 50px 10px;
+  width: calendar-width();
+
+  &__item {
+    display: flex;
+    width: $item-size;
+    height: $item-size;
+    border: 1px solid black;
+    background-color: #eee;
+    transition: transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+        display: flex;
+        width: 100%;
+        align-items: center;
+        justify-content: center;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
```

Comment:
It can be done on the element itself, you should keep the `content` in the pseudoelement only